lib/views/pr-detail-view.js

```diff
@@ -317,7 +270,7 @@ export class BareIssueishDetailView extends React.Component {
   }
 
   recordOpenInBrowserEvent() {
-    addEvent('open-issueish-in-browser', {package: 'github', from: 'issueish-header'});
+    addEvent('open-pull-request-in-browser', {package: 'github', component: this.constructor.name});
```

Comment:
Oh, cool, we get better granularity on these telemetry events, too.

Do we need to update anything or let @telliott27 know that this event name is changing?

Reply:
gooood question.  I was just thinking about that during our meeting this morning.  

@telliott27 : will this break the existing dashboard queries?